profiles/utilities.py

```diff
@@ -1,2 +1,14 @@
+import random
+import string
+
+
+def generate_id() -> str:
+    return generate_random_string(8, string.ascii_lowercase + string.digits)
```

Comment:
uppercase excluded for readability?

Reply:
8 should be a constant somewhere application-wide, as it's used in the database schema definition too

Reply:
Good idea. Not sure whether to pass in the generated ID, or move the `generate_random_string()` call into the object.

Reply:
Here I was referring only to the `8` length constant, but identity generation is serious business (as what characterizes all Entities in Domain-Driven Design) and could use a Factory